### ISSUE TITLE
Question about depth probing

### ISSUE THREAD

**Issue by caiocj1** [OpenHelix-Team/Spatial-Forcing#10]
Hello,

Thank you for the super interesting work! I have a question about depth probing.

I implemented spatial forcing (SF) on another repo and tried to reproduce depth probing there, so basically I'm taking the DPTHead from the VGGT repo and training it on the aligned hidden states and those of the first three layers per #7 . I'm separately doing the same thing then with the VLA without SF to compare. Is that how the comparison is done? Or is it just one DPTHead that is trained and then it makes inference on both VLAs?

Asking because in my current setup (2 separately trained DPTHeads) the one on the VLA w/ SF gets a lower depth loss which is expected but the depth maps from that and the baseline (without SF) are quite similar, the disparity qualitatively is not nearly as big as Figure 3 of the paper.

Any comment/insight would be appreciated, thanks.

**Comment by haofuly** [comment]
Hi @caiocj1, 

We use two separately trained DPTHeads. But the DPTHeads are trained from scratch rather than from VGGT pretrains. 

The observation that the one on the VLA w/ SF gets a lower depth loss is correct. 

I'm not sure which repo and dataset you implemented on. Could you try to train on datasetA (e.g., LIBERO-Spatial) and test on both datasetA (e.g., LIBERO-Spatial)  and datasetB (e.g., LIBERO-Long)  to examine the results?
